frontend/src/components/Header.js

```diff
@@ -19,7 +19,7 @@ export default function Header() {
   return (
     <div>
       <nav class="border-gray-200 bg-gray-50 dark:border-gray-700 dark:bg-gray-800">
-        <div class="mx-auto flex max-w-screen-xl flex-wrap items-center justify-between px-4 py-2 md:py-4">
+        <div class="mx-auto flex max-w-screen-xl flex-wrap items-center justify-between border-b border-gray-400 px-4 py-2 md:py-4">
```

Comment:
add a quick ss for this?

Reply:
![image](https://github.com/user-attachments/assets/c583f8a9-3dac-4fe0-b071-859773e25f89)
![image](https://github.com/user-attachments/assets/21afcda7-d4b9-4bc9-88a6-14aacbcee350)

Reply:
can probably add lines between the sections here also

Reply:
coolz.. lines between sections dont need i think.. enough space there